pkg/cloud/vsphere/provisioner/govmomi/create.go

```diff
@@ -322,58 +603,12 @@ func (pv *Provisioner) cloneVirtualMachineOnVCenter(s *SessionContext, cluster *
 	}
 	spec.PowerOn = true
 
-	if machineConfig.MachineSpec.VsphereCloudInit {
```

Comment:
Remove this field from machine spec?

Reply:
I wasn't going to remove any fields until we're in sync on templates and such.